guides/security/BYOD.md

```diff
@@ -0,0 +1,101 @@
+# Bring Your Own Device Policy
+
+This document highlights things you should think about, and how you manage your own device while working on Made Tech products and services. Not only the data you work with but also the software you have installed and how you protect it - like antivirus software.
+
+This only applies to you if you’re using your own device for Made Tech work - including checking your Made Tech email. Devices aren’t just desktop and laptops - they include, for example, tablets and smartphones.
+
+## Using and managing your device
+
+If you’re using your own device for Made Tech work, it means you’re being trusted to use Made Tech information appropriately and keep any Made Tech information or data you hold, or can access, safe.
+
+### Don’t ‘jailbreak’ your device
+‘Jailbreaking’ is the process of removing hardware and software vendor restrictions for the purpose of custom modifications (for example, installing unapproved applications or changing the user interface in an unsupported way).
+
+You must not use a jailbroken device for any Made Tech work or to handle any Made Tech data, including logging into Made Tech services which include (but is not limited to) G-Suite, Slack, password management software, etc.
+
+### Keep your software up to date
+You must keep your software up to date. Old versions of software are often vulnerable to attack. Update all the software on your device whenever a new version is available. This includes:
+
+- Your device’s operating system
+- Web browsers
+- Extensions
+- Plug-ins
+
+You must update your software within 14 days of a new verison being released.
+
+### Use endpoint protection software
+Unless your device is operating iOS or ChromeOS, your device must have endpoint protection software installed, which must also be kept up to date. 
+
+Endpoint protection is more than just antivirus software, and usually includes a firewall and may have other tools to protect your device, like network threat detection.
+
+Any device you plug into the device, like an external USB device, must also be scanned for viruses. We recommend doing a full scan at least once a month.
+
+### Set a password/passphrase
+Your device must require a password/passphrase to login, return from screensaver or wake from sleep. Ideally your device will lock the screen (or start the screensaver) after a few minutes (no more than 5) of the device being idle
```

Comment:
I think we can tighten this up to: your device will lock the screen (or start the screensaver) after no more than 2 minutes of the device being idle.